src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -122,3 +122,5 @@ export const selectFiatPriceHistoriesLoadingByTimeframe = createSelector(
   // if we don't have the data it's loading
   (fiatPriceHistory, currency, timeframe) => !Boolean(fiatPriceHistory[timeframe][currency]),
 )
+
+export const selectAllMarketData = (state: ReduxState) => state.marketData
```

Comment:
Why the new selector? We only use this for `enrichAssets` which accesses `marketData.crypto.byId[someId]`.
We can use `selectCryptoMarketData` which does exactly that, don't pass around the whole `marketData` slice, keep everything lighter (including tests) and revert changes in this file.